Makefile

```diff
@@ -18,8 +18,8 @@
         check-env check-env-config check-env-fragments \
         flat-headers \
         test \
-        install-headers install-libs install-lib-symlinks \
-        clean cleanmk cleanh cleanlib cleanleaves distclean \
+        install-headers install-libs install-lib-symlinks install-netlib-tests\
+        clean cleanmk cleanh cleanlib cleanleaves distclean clean-netlib-tests \
```

Comment:
The new netlibs test is built when building libFlame. It is also cleaned when a 'make clean' is run on libFlame.

As a note, the old test in netlibs-test/ was not removed because --enable-legacy-lapack still uses it.